### PR TITLE
chore: migrate project to schema version 80

### PR DESCRIPTION
**Pull request by kuldeep0020**
# Description

Migrate project to schema version 80.

## Linear Task
[Linear Ticket](https://linear.app/rudderstack/issue/PRO-3492/migrate-profiles-library-project-to-schema-version-80)

### CODE REVIEW

**Review by tipped73 — DISMISSED** [review]

File: models/localMacros.yaml [hunk 1]
```diff
@@ -2,27 +2,9 @@ macros:
...
-      {% if warehouse.DatabaseType() == \"bigquery\" %}
-        {% if !(end_time|isnil) %} date_diff(date('{{end_time.Format(\"2006-01-02 15:04:05\")}}'), date({{column}}), day)
-        {% else %} 
-        date_diff(CURRENT_DATE(), date({{column}}), day){% endif %}
-      {% else %}
-        {% if !(end_time|isnil) %} datediff(day, date({{column}}), date('{{end_time.Format(\"2006-01-02 15:04:05\")}}'))
-        {% else %} 
-        datediff(day, date({{column}}), GETDATE()) {% endif %}
-      {% endif %}"
+    value: " {% if warehouse.DatabaseType() == \"bigquery\" %} {% if !(end_time|isnil) %} date_diff(date('{{end_time.Format(\"2006-01-02 15:04:05\")}}'), date({{column}}), day) {% else %} date_diff(CURRENT_DATE(), date({{column}}), day){% endif %} {% else %} {% if !(end_time|isnil) %} datediff(day, date({{column}}), date('{{end_time.Format(\"2006-01-02 15:04:05\")}}')) {% else %} datediff(day, date({{column}}), GETDATE()) {% endif %} {% endif %}"
```
Comment (chandumlg) [inline models/localMacros.yaml:5]: Revert these changes?

Comment (kuldeep0020) [reply models/localMacros.yaml]: done
